shub/image/build.py

```diff
@@ -26,6 +26,7 @@
 """
 
 BUILD_STEP_REGEX = re.compile(r'Step (\d+)/(\d+) :*')
+BUILT_REGEX = re.compile(r'Successfully built ([0-9a-f]+)')
```

Comment:
While we're nitpicking anyways, I think the regex above this one is missing a `.` before the asterisk at the end. Unless you really wanted to match "an arbitrary number of colons" ;)